blocks/product-gallery-block/README.md

```diff
@@ -1,12 +1,13 @@
 # Product Gallery Block
 
-As a Page Builder user, I want to add the Product Gallery - Arc Block to my PDP page, so that I can merchandise multiple images on my PDP.
+As a Page Builder user, I want to add the Product Gallery - Arc Block to my PDP page, so that I can merchandise multiple images on my PDP. The Product Gallery uses the Carousel component in Themes Components, along with its indicator type functionality, to show options only available on the mobile viewport.
 
 ## Props
 
-| **Prop**               | **Required** | **Type** | **Description**                       |
-| ---------------------- | ------------ | -------- | ------------------------------------- |
-| isFeaturedImageEnabled | yes          | boolean  | Display the product’s featured image. |
+| **Prop**               | **Required** | **Type** | **Description**                                            |
+| ---------------------- | ------------ | -------- | ---------------------------------------------------------- |
+| isFeaturedImageEnabled | yes          | boolean  | Display the product’s featured image.                      |
+| indicatorType          | yes          | string   | Type of indicator to be displayed, like thumbnail or dots. |
```

Comment:
```suggestion
| indicatorType          | no          | string   | Type of indicator to be displayed, like thumbnail or dots. |
```

As this are custom fields they are not required

Reply:
ah right right